sorting_python.md

```diff
@@ -0,0 +1,111 @@
+# Sortings in Python
+### By: [Anmol Singh Yadav](https://github.com/IamLucif3r/)
+Below is compilation of every kind of sortings in Python
+
+The different implementations of sorting techniques in Python are:
+- Bubble Sort
+- Selection Sort
+- Insertion Sort
+
+## Bubble Sort
+Bubble Sort is a simple sorting algorithm. This sorting algorithm repeatedly compares two adjacent elements and swaps them if they are in the wrong order. It is also known as the sinking sort. It has a time complexity of O(n2) in the average and worst cases scenarios and O(n) in the best-case scenario. Bubble sort can be visualized as a queue where people arrange themselves by swapping with each other so that they all can stand in ascending order of their heights.
+
+### Bubble Sort Code
+
+```python
+# Python3 program for Bubble Sort Algorithm Implementation
+def bubbleSort(arr):
+	
+	n = len(arr)
+
+	# For loop to traverse through all
+	# element in an array
+	for i in range(n):
+		for j in range(0, n - i - 1):
+			
+			# Range of the array is from 0 to n-i-1
+			# Swap the elements if the element found
+			#is greater than the adjacent element
+			if arr[j] > arr[j + 1]:
+				arr[j], arr[j + 1] = arr[j + 1], arr[j]
+				
+arr = [ 2, 1, 10, 23 ]
+
+bubbleSort(arr)
+
+print("Sorted array is:")
+for i in range(len(arr)):
+	print("%d" % arr[i])
+
```

Comment:
You should use the bubble sort method here:
https://github.com/TheAlgorithms/Python/blob/master/sorts/bubble_sort.py